pybess/bess.py

```diff
@@ -221,15 +222,14 @@ def connect(self, grpc_url=DEF_GRPC_URL):
     # returns no error if already disconnected
     def disconnect(self):
         try:
-            if self.is_connected():
+            if self.channel is not None:
```

Comment:
I wonder if this is worth a comment - presumably the difference between `is_connected` and `channel is not None` is that `self.channel.unsubscribe` does something in the rare case that you have a channel but its status is not `grpc.ChannelConnectivity.READY`?

Reply:
Yeah, in such cases, it seemed that grpc keep retrying to connect in the background, with it own timeout. In the meantime, creating a new channel object doesn't do anything until the timeout fires. So I wanted to delete the channel so that a subsequent connection can be tried immediately.